### DISCUSSION TITLE
Welcome to hugo-theme-meme Discussions!

### DISCUSSION THREAD

**Discussion by reuixiy** [reuixiy/hugo-theme-meme#274] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ericpar** [comment]


**Reply by reuixiy** [reply]
Hi @ericpar, would you mind [create another new discussion](https://github.com/reuixiy/hugo-theme-meme/discussions/new) and categorize it into Q&A?

**Comment by ericpar** [comment]
Hi,

I'm a professional software developer (data engineering) and I am looking to start a blog on my hobbies, technologies and other topics that may come to my mind. I know very little about web UI so my questions may seem a little newbie so my apologies in advance. :)

I am new to Hugo but I'd like to learn from this platform; this is also a great opportunity to start learning about the Go programming language at the same time.

The Meme theme is super nice and provides me with all I ever wanted:
- multilingual
- minimalistic
- dark / light theme

Regards,

Eric
